pycbc/_version.py

```diff
@@ -28,6 +28,7 @@
 import logging
 
 logger = logging.getLogger('pycbc._version')
+SUPPRESS = "==SUPPRESS=="
```

Comment:
What is the point of this variable?

Reply:
Ah, thats a leftover from trying to combine argparse actions. I have deleted it now